examples/jsm/exporters/GLTFExporter.js

```diff
@@ -94,9 +94,17 @@ class GLTFExporter {
 	 * Parse scenes and generate GLTF output
 	 * @param  {Scene or [THREE.Scenes]} input   Scene or Array of THREE.Scenes
 	 * @param  {Function} onDone  Callback on completed
+	 * @param  {Function} onError  Callback on errors
 	 * @param  {Object} options options
 	 */
-	parse( input, onDone, options ) {
+	parse( input, onDone, onError, options ) {
+
+		// legacy: the third argument used to be options
+		if ( typeof onError === 'object' ) {
+
+			options = onError;
```

Comment:
Probably better to add a warning here? Something like:
```js
console.warn( 'THREE.GLTFExporter: parse() expects options as the fourth argument now.' );
```
This change also needs to be noted in the migration guide.

Reply:
Done.

> This change also needs to be noted in the migration guide.

I have no access to the wiki to make this change, can you take care of it?

Reply:
Yes!